tests/mocha/index.html

```diff
@@ -42,6 +42,8 @@
     <script src="connection_db_test.js"></script>
     <script src="connection_checker_test.js"></script>
     <script src="connection_test.js"></script>
+    <script src="connection_db_test.js"></script>
```

Comment:
Remove extra `connection_db_test.js`

Reply:
I think this was a merge conflict that I then messed up even further by removing connection_test instead of connection_db_test 🤦🏻‍♀️  However I think I've fixed it this time :)